authentication_service/example_config

```diff
@@ -0,0 +1,6 @@
+port: ":your_port_number"
```

Comment:
It does not have to be in yml

Reply:
No it just means that the config file will be taken as yml, it is required since the file has no extension. ofc if you have the extension, this line will not be required